input/cql/DRCommunicationWithPhysicianManagingDiabetesFHIR.cql

```diff
@@ -86,6 +90,9 @@ define "Qualifying Encounter During Measurement Period":
       and QualifyingEncounter.status = 'finished'
       and QualifyingEncounter.class !~ "virtual"
 
+/*
+@QICore6Uplift: Update Condition to ConditionProblemsHealthConcerns
+*/ 
```

Comment:
Should this be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis?